CheckFilelist.py

```diff
@@ -295,6 +295,14 @@ def ghostfile(pkg, f):
             '/etc/udev/rules.d/*',
         ],
     },
+    {
+        'error': 'suse-filelist-forbidden-xinetd-configuration',
+        'details': """Xinetd configuation files are deprecated.
+                      Please migrate to systemd unit files.""",
```

Comment:
is there some wiki page or so to link here that explains what needs to be done in more detail?